device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/AmqpsCbsReceiverLinkHandler.java

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package com.microsoft.azure.sdk.iot.device.transport.amqps;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.qpid.proton.amqp.messaging.Properties;
+import org.apache.qpid.proton.amqp.messaging.Released;
+import org.apache.qpid.proton.amqp.transport.DeliveryState;
+import org.apache.qpid.proton.engine.Delivery;
+import org.apache.qpid.proton.engine.Event;
+import org.apache.qpid.proton.engine.Receiver;
+
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+
+/**
+ * Every SAS token based authentication over AMQP requires a CBS session with a sender and receiver link. This
+ * class defines the receiver link which receives authentication status codes corresponding to each authentication attempt.
```

Comment:
Is this a new line or appears this way on github?

Reply:
I'm not sure that I see what you are seeing. This class javadoc is two lines for me

Reply:
I'm not sure I see what you are seeing